git/00.setup.md

```diff
@@ -66,37 +66,7 @@ alias gs="git status -sb"
 ## Git Status in Command Prompt
 
 Another useful part of git config is to have the branch names and status of the
-working directory expressed in your command prompt when in a git repo. Below is
-an example you can add to your `.bash_profile` as are starting point.
-
-```bash
-# in .bash_profile
-function _git_color() {
```

Comment:
I'm game, but if we do link to it, I think we should clean it up some (both explanation and code) and add more information about how to use it and what the output looks like. If you feel strongly about it, knock yourself out. Or I can take a whack at it if you'd like.